docs/checkit.js

```diff
@@ -1166,4 +1286,4 @@ CheckIt.prototype.opts = {
     shadow: false, // Whether to render a shadow
     hwaccel: false, // Whether to use hardware acceleration
     position: 'absolute' // Element positioning
-};
+};
```

Comment:
Missing a newline

Reply:
what do you mean?

Reply:
Because of the way that newline characters are used to indicate the end of the line it is expected that every file ends with a newline character.

Github showed a little warning that this file didn't have one (:no_entry_sign:).

Reply:
Like I have to go down to the bottom of the page and press enter?